apps/doc-gen/src/app/App.tsx

```diff
@@ -30,7 +30,7 @@ const App = () => {
   return (
     <div className="p-3">
       <h3>Compile a solidity contract in order to build documentation as markdown.</h3>
-      {fileName && <h6>File: {fileName.split('/')[1].split('.')[0].concat('.sol')}</h6>}
+      {fileName && <h6>File: {fileName.concat('.sol')}</h6>}
```

Comment:
why concat `.sol`. this doesn't look right

Reply:
I have since removed the concat. Thanks for pointing that out.